src/gengowatcher/website_setup.py

```
@@ -0,0 +1,221 @@
+    # Prompt for jobs_url
+    log("-" * 40)
+    log("Jobs Page URL")
+    log("-" * 40)
+    current_url = (
+        config.get("WebsiteMonitor", "jobs_url") or "https://gengo.com/t/jobs/"
+    )
+    log(f"Current: {current_url}")
+    log("The URL to monitor for new jobs. Usually: https://gengo.com/t/jobs/\n")
+
+    while True:
+        jobs_url = input(f"Enter jobs URL (Enter to keep current): ").strip()
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)